<launch>

 <node name="reasoning_node" pkg="roboy_mind" type="reasoning_node" cwd="node" output="screen" />

</launch>
